Delivered-To: mine260309@gmail.com
Received: by 2002:a4f:7f0c:0:0:0:0:0 with SMTP id a12csp60519ivd;
        Thu, 3 Jan 2019 13:56:36 -0800 (PST)
X-Google-Smtp-Source: AFSGD/XrMvujr2SqmuAKQkSGwvoDFom/spVuDODK30BhczoujpKzsWNgbRNjruRI+6IcTNCnGbkc
X-Received: by 2002:a62:cf84:: with SMTP id b126mr49373164pfg.98.1546552596300;
        Thu, 03 Jan 2019 13:56:36 -0800 (PST)
ARC-Seal: i=1; a=rsa-sha256; t=1546552596; cv=none;
        d=google.com; s=arc-20160816;
        b=i6F3sEB87HA4+iHbUZpCUWBwgCcNp2zmz0LZaoSVRmtvE+QBqXA9XFjUlXEJlNM1V/
         bhiDiqj0iH2eBDpwizzBS5o0HnklBoG3bZSW3WPCZz6G4eOvmCRQUR3SINZebXHSP+h5
         kXwOGGdOc6LGqa6vGAQ2zyLRiBDXeSAJdyRsFOrpkC/4vcqd0ts5724RlefLE2zduR2x
         22ynZ9TBJ8eFUJbj6dIFVX21yuOudo6TAOPqQSOz739eVUCEyVfKFeM0oUV20VDONU2p
         nU2IJl2eKVM7bRUsukSse2UHSo6Atwca6OfjSdOemVsMx8W3dhdca2gA2sL0zlePTEC5
         it/g==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=google.com; s=arc-20160816;
        h=sender:errors-to:cc:list-subscribe:list-help:list-post:list-archive
         :list-unsubscribe:list-id:precedence:message-id:references
         :in-reply-to:date:subject:to:from:delivered-to;
        bh=sAIPQcDIzFmxHqlLtk4BVyLk2WeZ1m4zfTrL0hTaJZ8=;
        b=k4cgG71RAA5lnUUTtEerV+l/MGks8iFjEDRZMZAp0zQQ/rsvL7uiG3yTfzMkF39dY5
         SwPeE3VgsNmQ1OrZ2vuD0hDQH208IVW8LJFCwhT2RFV1YMT8QV5q5F0MK6/OZD6LAnBA
         fIk3zlNPuJhg0bFSrnf4ww13XAOTkzMw51JTiEeFC6mLDwlh4E/XDWjBlMdJycmsti37
         8RQx4UE0CHBRLgKZl2CaWHVwNGkfFgMBcsA+npUYM+Tzdg2xnRGZ/KuH+F+k8gHRARhF
         Twej46G5jc48eGrRdOTyfOeUwPOOxNTyqORje/1sQNEiMnM2V8ZoEGdOp5UQAH9O5Sh1
         E+hQ==
ARC-Authentication-Results: i=1; mx.google.com;
       spf=pass (google.com: domain of openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org designates 203.11.71.2 as permitted sender) smtp.mailfrom="openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org";
       dmarc=fail (p=NONE sp=NONE dis=NONE) header.from=ibm.com
Return-Path: <openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org>
Received: from lists.ozlabs.org (lists.ozlabs.org. [203.11.71.2])
        by mx.google.com with ESMTPS id d4si47440151pls.348.2019.01.03.13.56.35
        for <mine260309@gmail.com>
        (version=TLS1_2 cipher=ECDHE-RSA-CHACHA20-POLY1305 bits=256/256);
        Thu, 03 Jan 2019 13:56:36 -0800 (PST)
Received-SPF: pass (google.com: domain of openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org designates 203.11.71.2 as permitted sender) client-ip=203.11.71.2;
Authentication-Results: mx.google.com;
       spf=pass (google.com: domain of openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org designates 203.11.71.2 as permitted sender) smtp.mailfrom="openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org";
       dmarc=fail (p=NONE sp=NONE dis=NONE) header.from=ibm.com
Received: from lists.ozlabs.org (lists.ozlabs.org [IPv6:2401:3900:2:1::3])
	by lists.ozlabs.org (Postfix) with ESMTP id 43W1w20S9WzDqGY
	for <mine260309@gmail.com>; Fri,  4 Jan 2019 08:56:34 +1100 (AEDT)
X-Original-To: openbmc@lists.ozlabs.org
Delivered-To: openbmc@lists.ozlabs.org
Authentication-Results: lists.ozlabs.org;
 spf=pass (mailfrom) smtp.mailfrom=linux.ibm.com
 (client-ip=148.163.156.1; helo=mx0a-001b2d01.pphosted.com;
 envelope-from=eajames@linux.ibm.com; receiver=<UNKNOWN>)
Authentication-Results: lists.ozlabs.org;
 dmarc=none (p=none dis=none) header.from=linux.ibm.com
Received: from mx0a-001b2d01.pphosted.com (mx0a-001b2d01.pphosted.com
 [148.163.156.1])
 (using TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits))
 (No client certificate requested)
 by lists.ozlabs.org (Postfix) with ESMTPS id 43W1sc4mPQzDqKw
 for <openbmc@lists.ozlabs.org>; Fri,  4 Jan 2019 08:54:28 +1100 (AEDT)
Received: from pps.filterd (m0098394.ppops.net [127.0.0.1])
 by mx0a-001b2d01.pphosted.com (8.16.0.22/8.16.0.22) with SMTP id
 x03LsNNB114228
 for <openbmc@lists.ozlabs.org>; Thu, 3 Jan 2019 16:54:25 -0500
Received: from e32.co.us.ibm.com (e32.co.us.ibm.com [32.97.110.150])
 by mx0a-001b2d01.pphosted.com with ESMTP id 2psrhvnfvu-1
 (version=TLSv1.2 cipher=AES256-GCM-SHA384 bits=256 verify=NOT)
 for <openbmc@lists.ozlabs.org>; Thu, 03 Jan 2019 16:54:24 -0500
Received: from localhost
 by e32.co.us.ibm.com with IBM ESMTP SMTP Gateway: Authorized Use Only!
 Violators will be prosecuted
 for <openbmc@lists.ozlabs.org> from <eajames@linux.ibm.com>;
 Thu, 3 Jan 2019 21:54:16 -0000
Received: from b03cxnp07029.gho.boulder.ibm.com (9.17.130.16)
 by e32.co.us.ibm.com (192.168.1.132) with IBM ESMTP SMTP Gateway: Authorized
 Use Only! Violators will be prosecuted; 
 (version=TLSv1/SSLv3 cipher=AES256-GCM-SHA384 bits=256/256)
 Thu, 3 Jan 2019 21:54:14 -0000
Received: from b03ledav003.gho.boulder.ibm.com
 (b03ledav003.gho.boulder.ibm.com [9.17.130.234])
 by b03cxnp07029.gho.boulder.ibm.com (8.14.9/8.14.9/NCO v10.0) with ESMTP id
 x03LsC4922216838
 (version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-GCM-SHA384 bits=256 verify=FAIL);
 Thu, 3 Jan 2019 21:54:12 GMT
Received: from b03ledav003.gho.boulder.ibm.com (unknown [127.0.0.1])
 by IMSVA (Postfix) with ESMTP id 9D0B86A04F;
 Thu,  3 Jan 2019 21:54:12 +0000 (GMT)
Received: from b03ledav003.gho.boulder.ibm.com (unknown [127.0.0.1])
 by IMSVA (Postfix) with ESMTP id 5D1776A051;
 Thu,  3 Jan 2019 21:54:12 +0000 (GMT)
Received: from talon7.ibm.com (unknown [9.41.179.222])
 by b03ledav003.gho.boulder.ibm.com (Postfix) with ESMTP;
 Thu,  3 Jan 2019 21:54:12 +0000 (GMT)
From: Eddie James <eajames@linux.ibm.com>
To: openbmc@lists.ozlabs.org
Subject: [PATCH dev-4.19 1/6] clk: aspeed: Add video engine reset index
 definition
Date: Thu,  3 Jan 2019 15:54:03 -0600
X-Mailer: git-send-email 1.8.3.1
In-Reply-To: <1546552448-22621-1-git-send-email-eajames@linux.ibm.com>
References: <1546552448-22621-1-git-send-email-eajames@linux.ibm.com>
X-TM-AS-GCONF: 00
x-cbid: 19010321-0004-0000-0000-000014CC6DA6
X-IBM-SpamModules-Scores: 
X-IBM-SpamModules-Versions: BY=3.00010339; HX=3.00000242; KW=3.00000007;
 PH=3.00000004; SC=3.00000272; SDB=6.01141311; UDB=6.00593987; IPR=6.00921536; 
 MB=3.00024957; MTD=3.00000008; XFM=3.00000015; UTC=2019-01-03 21:54:15
X-IBM-AV-DETECTION: SAVI=unused REMOTE=unused XFE=unused
x-cbparentid: 19010321-0005-0000-0000-00008A14BAD0
Message-Id: <1546552448-22621-2-git-send-email-eajames@linux.ibm.com>
X-Proofpoint-Virus-Version: vendor=fsecure engine=2.50.10434:, ,
 definitions=2019-01-03_10:, , signatures=0
X-Proofpoint-Spam-Details: rule=outbound_notspam policy=outbound score=0
 priorityscore=1501
 malwarescore=0 suspectscore=1 phishscore=0 bulkscore=0 spamscore=0
 clxscore=1015 lowpriorityscore=0 mlxscore=0 impostorscore=0
 mlxlogscore=999 adultscore=0 classifier=spam adjust=0 reason=mlx
 scancount=1 engine=8.0.1-1810050000 definitions=main-1901030187
X-BeenThere: openbmc@lists.ozlabs.org
X-Mailman-Version: 2.1.29
Precedence: list
List-Id: Development list for OpenBMC <openbmc.lists.ozlabs.org>
List-Unsubscribe: <https://lists.ozlabs.org/options/openbmc>,
 <mailto:openbmc-request@lists.ozlabs.org?subject=unsubscribe>
List-Archive: <http://lists.ozlabs.org/pipermail/openbmc/>
List-Post: <mailto:openbmc@lists.ozlabs.org>
List-Help: <mailto:openbmc-request@lists.ozlabs.org?subject=help>
List-Subscribe: <https://lists.ozlabs.org/listinfo/openbmc>,
 <mailto:openbmc-request@lists.ozlabs.org?subject=subscribe>
Cc: Eddie James <eajames@linux.ibm.com>
Errors-To: openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org
Sender: "openbmc" <openbmc-bounces+mine260309=gmail.com@lists.ozlabs.org>

Add an index into the array of resets kept in the Aspeed clock driver
for the video engine. This isn't a HW bit definition.

Signed-off-by: Eddie James <eajames@linux.ibm.com>
Acked-by: Stephen Boyd <sboyd@kernel.org>
Reviewed-by: Rob Herring <robh@kernel.org>
---
 include/dt-bindings/clock/aspeed-clock.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/dt-bindings/clock/aspeed-clock.h b/include/dt-bindings/clock/aspeed-clock.h
index f437386..15a9059 100644
--- a/include/dt-bindings/clock/aspeed-clock.h
+++ b/include/dt-bindings/clock/aspeed-clock.h
@@ -50,5 +50,6 @@
 #define ASPEED_RESET_I2C		7
 #define ASPEED_RESET_AHB		8
 #define ASPEED_RESET_CRT1		9
+#define ASPEED_RESET_VIDEO		10
 
 #endif
-- 
1.8.3.1

